packages/api/tests/endpoints/collections/delete-collection.js

```diff
@@ -386,112 +305,6 @@ test.serial('Attempting to delete a collection with an associated rule does not
   ));
 });
 
-test.serial('del() does not remove from Elasticsearch or publish SNS message if removing from PostgreSQL fails', async (t) => {
```

Comment:
This test might still be valid without ES? e.g. `del() does not publish SNS message if removing from PostgreSQL fails`